install_ubuntu.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+GREEN='\033[39;42m'
+NOCOLOR='\033[0m'
+
+sudo wget -O /usr/local/bin/ufw-docker \
+  https://github.com/chaifeng/ufw-docker/raw/master/ufw-docker
+sudo chmod +x /usr/local/bin/ufw-docker
+ufw-docker install
+ufw-docker status
+
+sudo ufw reload
```

Comment:
Can we just delete the ufw-docker script at this point? It is not needed anymore.